Sources/FluidAudio/Diarizer/Extraction/EmbeddingExtractor.swift

```diff
@@ -3,38 +3,14 @@ import CoreML
 import OSLog
 
 /// Embedding extractor with ANE-aligned memory and zero-copy operations
-public class EmbeddingExtractor {
+public final class EmbeddingExtractor {
     private let wespeakerModel: MLModel
     private let logger = AppLogger(category: "EmbeddingExtractor")
     private let memoryOptimizer = ANEMemoryOptimizer()
 
-    // Pre-allocated ANE-aligned buffers
-    private var waveformBuffer: MLMultiArray?
```

Comment:
no impact and a lot of concurrency issues arise from this


main RTFx `305.2` this branch RTFx `299.5`